### DISCUSSION TITLE
how to hightlight empty gird area when i dragging items

### DISCUSSION THREAD

**Discussion by Asif-Saheer-k** [react-grid-layout/react-grid-layout#2006] [General]
In want to highlight the empty grid areas when dragging an item, I can dynamically add a placeholder element that represents the empty space
![image](https://github.com/react-grid-layout/react-grid-layout/assets/100151856/30dc96be-e6fa-4335-a08f-5fe5210a7348)
like this image

**Comment by dv1651st** [comment]
you can, you just need to get the placeholders position inside of grid layout. unfortunately, the ondrag and onresize item callbacks are jank, and so you kind of have to settle for a jank solution. use an effect hook, set an interval to repeat every 100ms or so, and then use query selector to select the element with the ".react-grid-item.react-grid-placeholder" class. that'll let you get the elements top and left position which, combined with height and width, will allow other components to in your app to know where the placeholder is, incase you are using a custom background. that should get you where you need.
